bootstrap/eks/internal/userdata/nodeadm.go

```diff
@@ -115,22 +115,6 @@ spec:
 {{ Indent 6 (toYaml .ContainerdBaseRuntimeSpec) }}
     {{- end }}
   {{- end }}
-  {{- if .Instance }}
-  instance:
-    {{- if .Instance.LocalStorage }}
-    localStorage:
-      strategy: {{ .Instance.LocalStorage.Strategy }}
-      {{- with .Instance.LocalStorage.MountPath }}
-      mountPath: {{ . }}
```

Comment:
Why are we removing it here instead of adding "Local mount paths" in the NodeadmConfig type?